### PR TITLE
Pull the Failing -> Degraded rename + OpenID IdP API change

### PR DESCRIPTION
**Pull request by stlaz**
Keeping up with API changes

### CODE REVIEW

File: pkg/operator2/idp.go [hunk 1]
```diff
@@ -233,16 +233,7 @@ func (c *authOperator) convertProviderConfigToIDPData(providerConfig *configv1.I
...
-func (c *authOperator) discoverOpenIDURLs(issuer, key string, ca configv1.ConfigMapNameReference, fallbackURLs configv1.OpenIDURLs) (*osinv1.OpenIDURLs, error) {
-	// TODO drop after 4.0beta3
-	if len(issuer) == 0 {
-		return &osinv1.OpenIDURLs{
-			Authorize: fallbackURLs.Authorize,
-			Token:     fallbackURLs.Token,
-			UserInfo:  fallbackURLs.UserInfo,
-		}, nil
-	}
-
```
Comment (sallyom) [inline pkg/operator2/idp.go]: I _think_ this should be good now, since we're beta4

**Comment by stlaz** [comment]
Unauthorized :octopus: 
/retest

File: pkg/operator2/idp.go (hunk 1 above)
Comment (enj) [reply pkg/operator2/idp.go]: I believe this is fine now.

**Pull request merged**